### DISCUSSION TITLE
No entry point for custom callbacks?

### DISCUSSION THREAD

**Discussion by einarbmag** [explosion/spaCy#7456] [Help: Coding & Implementations] [answered]
I'm customizing my tokenizer exactly as described in https://spacy.io/usage/training#custom-tokenizer

I want to put any reusable code like this in its own package for use within my team. But it looks like it isn't possible to register callback functions using entry points? Why not? Any alternatives?

**Comment by adrianeboyd** [comment, marked as answer]
Hmm, that looks like an oversight. In the meanwhile, you can use `misc` instead of `callbacks`.

**Comment by einarbmag** [comment]
Ah ok, great, thanks!
